cfn-resources/alert-configuration/cmd/resource/resource.go

```diff
@@ -300,20 +300,20 @@ func expandAlertConfigurationMetricThresholdConfig(currentModel *Model) *atlasSD
 	}
 }
 
-func expandAlertConfigurationThreshold(threshold *IntegerThresholdView) *atlasSDK.GreaterThanRawThreshold {
+func expandAlertConfigurationThreshold(threshold *IntegerThresholdView) *admin.GreaterThanRawThreshold {
 	if threshold == nil {
 		return nil
 	}
-	return &atlasSDK.GreaterThanRawThreshold{
+	return &admin.GreaterThanRawThreshold{
 		Operator:  threshold.Operator,
 		Threshold: util.Pointer(int(*threshold.Threshold)),
```

Comment:
this seems a bit overcomplicated

Reply:
It does have the challenge of passing from float64 to int (CFN model defines as float64). This does raise a bigger point that we might be losing precision for this case, will take the action of seeing what type is in mms and if a fix is need in the SDK to be able to pass float values.